docs-src/references/dynamic-configuration.md

```diff
@@ -0,0 +1,209 @@
+    value: 20 
+  - constraints: {namespace: "namespace2"}  # Applies to all task queues in "namespace2".
+    value: 10
+  - constraints: {}  # Applies to all other task queues in "namespace1" and all other Namespaces.
+    value: 16
+  ```
+
+<!-- Note that the values set with most constraints take priority over values that are set with fewer constraints, regardless of the order in which they are set in the dynamic configuration key. -->
+
+For more examples on how dynamic configuration is set, see:
+
+- [docker-compose](https://github.com/temporalio/docker-compose/tree/main/dynamicconfig)
+- [samples-server](https://github.com/temporalio/samples-server/blob/main/tls/config/dynamicconfig/development.yaml)
+
+## Commonly used dynamic configuration keys
+
+The following table lists commonly used dynamic configuration keys that can be used for rate limiting requests to the Temporal Cluster.
+
+Note that the dynamic configuration key setting is optional. If you choose to update these values for your Temporal Cluster, ensure that you are provisioning enough resources to handle the load.
+
+All values listed here are for Temporal server v1.20 <!--Update to 1.21 when released-->.
+Check [server release notes](https://github.com/temporalio/temporal/releases) to verify any potential breaking changes when upgrading your versions.
+
+### Service-level RPS limits
+
+The Rate Per Second (RPS) dynamic configuration keys set the rate at which requests can be made to each service in your Cluster.
+
+When scaling your services, tune the RPS to test your workload and set acceptable provisioning benchmarks. Exceeding these limits will result in `ResourceExhaustedError`.
+
+| Dynamic configuration key              | Type | Description                                                                                                                                                                                                                        | Default value |
+| -------------------------------------- | ---- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| Frontend                               |      |                                                                                                                                                                                                                                    |               |
+| `frontend.rps`                         | Int  | Rate limit per second on the number of requests send to each Frontend service host.                                                                                                                                                | 2400          |
```

Comment:
```suggestion
| `frontend.rps`                         | Int  | Rate limit (requests/second) for requests accepted by each Frontend service host.                                                                                                                                                | 2400          |
```
and similar changes on the others

i.e. the dynamic config doesn't control requests _sent_ to a frontend, that's determined by whatever is sending it traffic. the config controls how many the frontend _accepts_ (the rest get rejected with a "resource exhausted" error)

Reply:
Also: In "Frontend service", capitalize "service".

Reply:
ty; correcting this across